src/sentry/runner/commands/backup.py

```diff
@@ -38,7 +39,7 @@ def pretty(self) -> str:
 class ComparatorFinding(NamedTuple):
     """Store all information about a single failed matching between expected and actual output."""
 
-    name: ComparatorName
+    name: ComparatorSlug
```

Comment:
`slug:`?

Reply:
Changed to `kind` and `ComparatorKind`, respectively, for clarity.